RELEASE.rst

```diff
@@ -0,0 +1,6 @@
+RELEASE_TYPE: patch
+
+This release fixes an issue where the decorator used to rename arguments could
+throw a ``TypeError`` if used on a function without a docstring.  This was
```

Comment:
We should update to clarify that this would manifest on `-O2` or greater.

Reply:
Yep.